index.html

```diff
@@ -69,6 +69,11 @@
               <li><a href="http://kaunas-jug.lt">Kaunas JUG</a></li>
             </ol>
         </div>
+        <div class="row">
+	    <a href="http://www.sourcecodepoetry.com/" target="_blank">
+	        <img src="images\source_code_poetry_shakespeare400.png" alt="Source Code Poetry">
```

Comment:
Please fix slashes for image URL. I would recommend to put banner before UG list.